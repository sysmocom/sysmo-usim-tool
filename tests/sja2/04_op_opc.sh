#!/bin/sh
. ./test-data

# OP/OPc is milenage specific, make sure we have milenage configured before
# testing.
$TOOL -a $ADMPIN -T "MILENAGE:MILENAGE"

$TOOL -a $ADMPIN -C 000102030405060708090a0b0c0d0e0f
$TOOL -a $ADMPIN -o

$TOOL -a $ADMPIN -O 840337c3d45397ce8ea8609ffdc47224
$TOOL -a $ADMPIN -o

