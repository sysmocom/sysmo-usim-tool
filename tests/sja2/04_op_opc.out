sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Programming Authentication parameters...
 * Initializing...
 * New algorithm setting:
   2g: 4=MILENAGE
   3g: 4=MILENAGE
   4g5g: 4=MILENAGE
 * Programming...

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Writing OPc value...
 * Initializing...
 * New OPc setting:
   OPc: 000102030405060708090a0b0c0d0e0f
 * Programming...
   2g OPc programmed.
   3g OPc programmed.
   4g5g OPc programmed.

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Reading OP/c value...
 * Initializing...
 * Reading...
 * Current OP/OPc setting:
   2g: OPc: 000102030405060708090a0b0c0d0e0f
   3g: OPc: 000102030405060708090a0b0c0d0e0f
   4g5g: OPc: 000102030405060708090a0b0c0d0e0f

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Writing OP value...
 * Initializing...
 * New OPc setting:
   OP: 840337c3d45397ce8ea8609ffdc47224
 * Programming...
   2g OP programmed.
   3g OP programmed.
   4g5g OP programmed.

Done!
sysmoISIM-SJA2 parameterization tool
Copyright (c) 2019-2022 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 4C 75 30 34 05 4B A9
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Reading OP/c value...
 * Initializing...
 * Reading...
 * Current OP/OPc setting:
   2g: OP: 840337c3d45397ce8ea8609ffdc47224
   3g: OP: 840337c3d45397ce8ea8609ffdc47224
   4g5g: OP: 840337c3d45397ce8ea8609ffdc47224

Done!
